drop-in/src/main/java/com/adyen/checkout/dropin/ui/paymentmethods/PaymentMethodAdapter.kt

```diff
@@ -28,20 +32,36 @@ import com.adyen.checkout.dropin.ui.paymentmethods.PaymentMethodListItem.Compani
 
 @SuppressWarnings("TooManyFunctions")
 class PaymentMethodAdapter(
-    private val paymentMethods: List<PaymentMethodListItem>,
-    private val imageLoader: ImageLoader
+    private val paymentMethods: MutableList<PaymentMethodListItem>,
+    private val imageLoader: ImageLoader,
+    private val onUnderlayExpandListener: ((AdyenSwipeToRevealLayout) -> Unit)? = null
```

Comment:
Is this `onUnderlayExpandListener` ever being used? Why do we need to know if the underlay was expanded in this use case?